File: JavaScript/05_battery_low_beginner.js
const readline = require("readline");

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});

rl.question("", (t) => {
  const tests = parseInt(t);
  rl.on("line", (line) => {
    const x = parseInt(line);
    if (x <= 15) {
      console.log("Yes");
    } else {
      console.log("No");
    }
  });
});

/**********************************************************
 * @INFO
 * Code by Ashish Singh
 * @INFO
 * Gmail - ashishlodhi5559@gmail.com
 * @INFO
 * Github - AshishBytes
 * @INFO
 * LinkedIn - ashishbytes
 * @INFO
 * Instagram - @itz_ash._u
 * @INFO
 * Twitter - @itz_ash_u
 * @INFO
 **********************************************************/

File: Golang/05_battery_low_beginner.go
package main

import "fmt"

func main() {
	var t, x int
	fmt.Scan(&t)
	for i := 0; i < t; i++ {
		fmt.Scan(&x)
		if x <= 15 {
			fmt.Println("Yes")
		} else {
			fmt.Println("No")
		}
	}
}

/**********************************************************
 * @INFO
 * Code by Ashish Singh
 * @INFO
 * Gmail - ashishlodhi5559@gmail.com
 * @INFO
 * Github - AshishBytes
 * @INFO
 * LinkedIn - ashishbytes
 * @INFO
 * Instagram - @itz_ash._u
 * @INFO
 * Twitter - @itz_ash_u
 * @INFO
 **********************************************************/
